smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
I guess what I'm not getting is why are they being included in the first place if they aren't valid? Are we including them automagically somewhere? Shouldn't it just be the fields specified in `fields` for the view?